### PR TITLE
Site cake config

### PR DESCRIPTION
**Pull request by lukasrzavsky**

### CODE REVIEW

File: src/components/molecules/CakeOptions.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  display: 'flex',
+  alignItems: 'left',
+  flexDirection: 'column',
+  [mobileQuery]: { marginLeft: '8%', marginBottom: '41px', width: '90%' },
+  [tabletQuery]: { marginLeft: '10%', marginBottom: '41px', width: '60%' },
+  [laptopQuery]: { marginLeft: '15%', width: '80%' },
+  [desktopQuery]: { margin: '0 auto 41px 15%', width: '80%' },
+});
+
+const UnorderList = styled.ul({
```
Comment (MattGiertl) [inline src/components/molecules/CakeOptions.js]: create a <List /> component out of this and move it to atoms

File: src/components/molecules/CakeType.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }),
+);
+
+const UnorderList = styled.ul({
+  listStyleType: 'none',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'flex-end',
+  paddingInlineStart: '0px',
```
Comment (MattGiertl) [inline src/components/molecules/CakeType.js]: what does this do? is it needed?

File: src/components/molecules/CakeType.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+      marginTop="19px"
+    >
+      {type}
+    </Paragraph>
+    <UnorderList>
+      <ListItem>{weight}</ListItem>
+      <ListItem>{height}</ListItem>
+      <ListItem>{portions}</ListItem>
+      <ListItem marginTop="20px">{price}</ListItem>
+    </UnorderList>
```
Comment (MattGiertl) [inline src/components/molecules/CakeType.js]: use the list component.. also, this is gramatically incorrect

File: src/data/cakeConfig.json [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+          "image": "/assets/MidiCake.png",
+          "imgName": "Midi",
+          "type": "Midi",
+          "weight": "2000 g",
+          "height": "výška 13 cm",
+          "portions": "10 porcií",
+          "price": "od 55€"
+        },
+        {
+          "image": "/assets/KlasikCake.png",
```
Comment (MattGiertl) [inline src/data/cakeConfig.json]: ClassicCake

File: src/pages/torty-na-mieru.js [hunk 5]
```diff
@@ -0,0 +1,173 @@
...
+    const renderSecondCardItems = secondCard.items.map(item => (
+      <CakeOptions src={item.image} imgName={item.imgName} heading={item.heading}>
+        {item.options.map(item => (
+          <ListItem>{item}</ListItem>
+        ))}
+      </CakeOptions>
+    ));
+
+    return (
+      <>
```
Comment (MattGiertl) [inline src/pages/torty-na-mieru.js:60]: useless fragment here?

File: src/templates/CakeConfigTemplate.js [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+            lineHeight="24px"
+          >
+            <SocialButton
+              to="https://www.instagram.com/makronkovo/?hl=en"
+              marginRight="12px"
+              icon={['fab', 'instagram']}
+            />
+            makronkovo
+          </Anchor>
+          <Anchor
```
Comment (MattGiertl) [inline src/templates/CakeConfigTemplate.js:74]: target: '_blank' on the Anchor

Comment (MattGiertl) [reply src/templates/CakeConfigTemplate.js:74]: if it isn't already in the component :P

**Review by MattGiertl — CHANGES_REQUESTED** [review]

File: src/pages/torty-na-mieru.js (hunk 5 above)
Comment (lukasrzavsky) [reply src/pages/torty-na-mieru.js:60]: Its needed, because there are multiple components

File: src/components/molecules/CakeType.js (hunk 2 above)
Comment (lukasrzavsky) [reply src/components/molecules/CakeType.js]: Its needed because if the paddingInlineStart has some different value there is an indentation of the items

**Review by MattGiertl — DISMISSED** [review]

**Review by MattGiertl — APPROVED** [review]

**Pull request merged**